src/layoutcomponent/Navbar.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { Logo } from "./Logo";
+import { NavbarSearch } from "./NavbarSearch";
+import { HamburgerSearch } from "./HamburgerSearch";
+import "../styles/Navbar.css";
+import { UserProfileInNavbar } from "./UserProfileInNavbar";
+import { HamburgerNavbar } from "./HamburgerNavbar";
+import { useState } from "react";
```

Comment:
useState can be imported in first line along with React as these are coming from same module

Reply:
okey